kratos/utilities/os_utilities.cpp

```diff
@@ -10,10 +10,15 @@
 //  Main authors:    Vicente Mataix Ferrandiz
 //
 
+// Project includes
+#include "utilities/os_utilities.h"
```

Comment:
```suggestion
#include "utilities/os_utilities.h" # has to be included before using KRATOS_COMPILED_IN_WINDOWS
```

to prevent this from being moved in the future